src/cloud-api-adaptor/Makefile

```diff
@@ -199,6 +199,12 @@ podvm-binaries:
 	$(DOCKER_OPTS) .
 	rm -rf .git
 
+# when local image is used it needs to be mounted in the container
+ifdef IMAGE_LOCAL_PATH
+IMAGE_OPTS = --build-arg IMAGE_CHECKSUM=$(IMAGE_CHECKSUM) --build-arg IMAGE_URL=/tmp/image.img -v $(IMAGE_LOCAL_PATH):/tmp/image.img:Z
```

Comment:
does docker buildx allows volume mount during build ? Last I checked `-v` was not available with docker buildx

Reply:
Right, thanks will switch to COPY